aea/configurations/project.py

```diff
@@ -61,10 +62,13 @@ def load(
         ctx.set_config("skip_consistency_check", skip_consistency_check)
         path = os.path.join(working_dir, public_id.author, public_id.name)
         target_dir = os.path.join(public_id.author, public_id.name)
-        if is_local:
-            fetch_agent_locally(ctx, public_id, target_dir=target_dir)
-        else:
-            fetch_agent(ctx, public_id, target_dir=target_dir)
+
```

Comment:
`path` has to be checked with os.path.exists

Reply:
It's checked in `fetch_agent` and if we trying to fetch new project (not to restore) that will fail with an error which is an expected behavior.

Reply:
If it's absent for some reason but `is_restore == True` we proceed to fetch it.